docs/fundamentals/decision-guide-pipeline-dataflow-eventstream-spark.md

```diff
@@ -1,31 +1,31 @@
+Use this reference guide and the example scenarios to help you in deciding whether you need a copy activity, a dataflow, an Eventstream, or Spark for your Microsoft Fabric workloads.
 
-## Copy activity, dataflow, and Spark properties
+## Copy activity, dataflow, Eventstream, and Spark properties
 
-| | **Pipeline copy activity** | **Dataflow Gen 2** | **Spark** |
-|---|:---:|:---:|:---:|
-| **Use case** | Data lake and data warehouse migration,<br>data ingestion,<br>lightweight transformation | Data ingestion,<br>data transformation,<br>data wrangling,<br>data profiling | Data ingestion,<br>data transformation,<br>data processing,<br>data profiling |
-| **Primary developer persona** | Data engineer,<br>data integrator | Data engineer,<br>data integrator,<br>business analyst | Data engineer,<br>data scientist,<br>data developer |
-| **Primary developer skill set** | ETL,<br>SQL,<br>JSON | ETL,<br>M,<br>SQL | Spark (Scala, Python, Spark SQL, R) |
-| **Code written** | No code,<br>low code | No code,<br>low code | Code |
-| **Data volume** | Low to high | Low to high | Low to high |
-| **Development interface** | Wizard,<br>canvas | Power query | Notebook,<br>Spark job definition |
-| **Sources** | 30+ connectors | 150+ connectors | Hundreds of Spark libraries |
-| **Destinations** | 18+ connectors | Lakehouse,<br>Azure SQL database,<br>Azure Data explorer,<br>Azure Synapse analytics | Hundreds of Spark libraries |
-| **Transformation complexity** | Low:<br>lightweight - type conversion, column mapping, merge/split files, flatten hierarchy | Low to high:<br>300+ transformation functions | Low to high:<br>support for native Spark and open-source libraries |
+| | **Pipeline copy activity** | **Dataflow Gen 2** | **Eventstream** | **Spark** |
+|---|:---:|:---:|:---:|:---:|
+| **Use case** | Data lake and data warehouse migration,<br>data ingestion,<br>lightweight transformation | Data ingestion,<br>data transformation,<br>data wrangling,<br>data profiling | event data ingestion,<br>event data transformation | Data ingestion,<br>data transformation,<br>data processing,<br>data profiling |
+| **Primary developer persona** | Data engineer,<br>data integrator | Data engineer,<br>data integrator,<br>business analyst | Data engineer,<br>data scientist,<br>data developer | Data integrator,<br>data engineer |
+| **Primary developer skill set** | ETL,<br>SQL,<br>JSON | ETL,<br>M,<br>SQL | SQL, JSON, messaging |Spark (Scala, Python, Spark SQL, R) |
+| **Code written** | No code,<br>low code | No code,<br>low code | No Code, <br>low code | Code |
+| **Data volume** | Low to high | Low to high | Medium to High | Low to high |
+| **Development interface** | Wizard,<br>canvas | Power query | Canvas | Notebook,<br>Spark job definition |
+| **Sources** | 30+ connectors | 150+ connectors | Database CDC, Kafka, Pub/Sub Messaging Systems, Event streams | Hundreds of Spark libraries |
```

Comment:
It would be nice to have a link to the following article that lists all the supported sources. 
https://learn.microsoft.com/en-us/fabric/real-time-intelligence/event-streams/add-manage-eventstream-sources?pivots=enhanced-capabilities#supported-sources

Please remove "Event streams" from the list of sources.